ultravox/training/configs/meta_config.yaml

```diff
@@ -1,4 +1,4 @@
-text_model: "meta-llama/Meta-Llama-3-8B-Instruct"
+text_model: "meta-llama/Meta-Llama-3.1-8B-Instruct"
```

Comment:
Changing default. This might be a bit pre-mature, so I don't have a strong opinion on it.

Reply:
Maybe keep llama3 for now and consider switching to llama3.1 after running more experiments?

Reply:
Okay reverting.